tests/back/dataloaders/test_dashai_dataset.py

```diff
@@ -147,24 +147,17 @@ def test_save_to_disk_and_load():
     inputs_columns = separate_dataset["train"].inputs_columns
     outputs_columns = separate_dataset["train"].outputs_columns
 
-    for i in separate_dataset.keys():
-        separate_dataset[i].save_to_disk(f"tests/back/dataloaders/dashaidataset_{i}")
-    paths = {
-        "path_train": "tests/back/dataloaders/dashaidataset_train",
-        "path_val": "tests/back/dataloaders/dashaidataset_validation",
-        "path_test": "tests/back/dataloaders/dashaidataset_test",
-    }
-
-    dashaidataset_train = load(paths["path_train"])
-    dashaidataset_val = load(paths["path_val"])
-    dashaidataset_test = load(paths["path_test"])
-    dashai_datasetdict = DatasetDict(
-        {
-            "train": dashaidataset_train,
-            "validation": dashaidataset_val,
-            "test": dashaidataset_test,
-        }
-    )
+    save_dataset(separate_dataset, "tests/back/dataloaders/dashaidataset")
+
+    # for i in separate_dataset.keys():
+    #     separate_dataset[i].save_to_disk(f"tests/back/dataloaders/dashaidataset_{i}")
+    # paths = {
+    #     "path_train": "tests/back/dataloaders/dashaidataset_train",
+    #     "path_val": "tests/back/dataloaders/dashaidataset_validation",
+    #     "path_test": "tests/back/dataloaders/dashaidataset_test",
+    # }
```

Comment:
Delete commented code when you are pushing to staging. 
Otherwise, we tend to fill up with commented code that we don't know what it does or why it is there 😥.
```suggestion
```

Reply:
My bad, i forgot to delete it